mapswipe_workers/mapswipe_workers/utils/user_management.py

```diff
@@ -158,3 +158,52 @@ def permission_denied(request_object):
             return True
         else:
             raise HTTPError(e, request_object.text)
+
+
+def add_team(email, team_id):
```

Comment:
I find this function name a bit confusing, I expect it to create a new team, just from the name. Could you rename it to something like `add_user_to_team`? Same thought about the function `remove_team` below.

Reply:
agreed